src/gui/torrentmodel.h

```diff
@@ -101,6 +102,7 @@ private slots:
 
 private:
     QList<BitTorrent::TorrentHandle *> m_torrents;
+    QObject *m_parent;
```

Comment:
1. `const` object is sufficient here.
2. What is the point of declaring it as `QObject` to always `dynamic_cast` it to its actual type on use? Why don't you declare it as `const TransferListWidget`?